docs/README.md

```diff
@@ -2,27 +2,19 @@
 
 Kubernetes allows for extensions to its architecture in the form of custom
 resources: <https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/>.
-KubeVirt represents virtual machines as custom resources and manages changes
-to libvirt domains based on the state of those resources.
+KubeVirt represents virtual machines as custom resources and manages changes to libvirt domains based on the state of those resources.
 
 ### Project Components
 
- * virt-api: This component provides a HTTP RESTful entrypoint to manage
-   the virtual machines within the cluster.
- * virt-controller: This component manages the state of each VMI within the
-   Kubernetes cluster.
- * virt-handler: This is a daemon that runs on each Kubernetes node. It is
-   responsible for monitoring the state of VMIs according to Kubernetes and
-   ensuring the corresponding libvirt domain is booted or halted accordingly.
- * virt-launcher: This component is a place-holder, one per running VMI. It
-   contains the running VMI and remains running as long as the VMI is defined.
+ * virt-api: This component provides a HTTP RESTful entrypoint to manage the virtual machines within the cluster.
```

Comment:
here, two, I fail to identify why the change is needed.

Reply:
ok, i did one commit for the changes of each component description....